sdks/java/core/src/main/java/org/apache/beam/sdk/io/FileIO.java

```diff
@@ -262,21 +262,21 @@
  *     writer.println(Joiner.on(",").join(element));
  *   }
  *
- *   public void finish() throws IOException {
+ *   public void flush() throws IOException {
  *     writer.flush();
  *   }
  * }
  *
  * PCollection<BankTransaction> transactions = ...;
  * // Convert transactions to strings before writing them to the CSV sink.
  * transactions.apply(MapElements
- *         .into(lists(strings()))
+ *         .into(TypeDescriptors.lists(TypeDescriptors.strings()))
```

Comment:
The existing example assumed a static import of `TypeDescriptors`. It seems like it would be difficult for new Beam users to infer this, so better to use `TypeDescriptors` verbosely here.

Reply:
+1